README.md

```diff
@@ -148,10 +148,10 @@ Guidelines:
 
 To prevent accidental violation of one of those guidelines, you should *never*
 
-* [learn git](http://git-scm.com/book/en/Git-Basics)
+* [learn git](https://git-scm.com/book/en/v1/Git-Basics)
```

Comment:
I'd rather leave this without the version, then it should redirect to the latest version.